From: NIIBE Yutaka <gniibe@fsij.org>
Date: Tue, 18 Oct 2022 10:16:11 +0900
Subject: dirmngr: Fix build with no LDAP support.

* dirmngr/server.c [USE_LDAP] (start_command_handler): Conditionalize.

--

Cherry-pick master commit of:
	7011286ce6e1fb56c2989fdafbd11b931c489faa

GnuPG-bug-id: 6239
Signed-off-by: NIIBE Yutaka <gniibe@fsij.org>
(cherry picked from commit a5c3821664886ffffbe6a83aac088a6e0088a607)
---
 dirmngr/server.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/dirmngr/server.c b/dirmngr/server.c
index 651f67c..87a0d77 100644
--- a/dirmngr/server.c
+++ b/dirmngr/server.c
@@ -3135,8 +3135,10 @@ start_command_handler (assuan_fd_t fd, unsigned int session_id)
                ctrl->refcount);
   else
     {
+#if USE_LDAP
       ks_ldap_free_state (ctrl->ks_get_state);
       ctrl->ks_get_state = NULL;
+#endif
       release_ctrl_ocsp_certs (ctrl);
       xfree (ctrl->server_local);
       dirmngr_deinit_default_ctrl (ctrl);
